src/routes/admin/parts/[partId]/+page.server.js

```diff
@@ -1,6 +1,7 @@
 import {getUserRoles} from "$lib/server/account.js";
 import {pool} from "../../../../hooks.server.js";
 import {redirect} from "@sveltejs/kit";
+import {getPartInfo, updatePart} from "../../../../lib/server/parts.js";
```

Comment:
On peut aussi raccourcir l'import ici.
```suggestion
import {getPartInfo, updatePart} from '$lib/server/parts.js';
```